en/Plugins/Releasing/Plugin guidelines.md

```diff
@@ -376,3 +376,28 @@ async function AsyncTest(): Promise<string | null> {
   }
 }
 ```
+
+## Language
+
+### English-first
+
+Write the following in English by default:
+
+- `id` property in `manifest.json`
+- `name` property in `manifest.json`
+- `description` property in `manifest.json`
```

Comment:
Maybe indent these, so it has the same structure?
- `manifest.json`
   -  `id`
   - `name`
   - `description`
- `README.md`